examples/mplot3d/contour3d_demo3.py

```diff
@@ -1,20 +1,34 @@
+'''
+Demonstrates displaying a 3D surface while also projecting contour 'profiles'
+onto the 'walls' of the graph.
+
+See contourf3d_demo2 for the filled version.
+'''
+
 from mpl_toolkits.mplot3d import axes3d
 import matplotlib.pyplot as plt
 from matplotlib import cm
 
 fig = plt.figure()
 ax = fig.gca(projection='3d')
 X, Y, Z = axes3d.get_test_data(0.05)
+
+# Plot the 3D surface
 ax.plot_surface(X, Y, Z, rstride=8, cstride=8, alpha=0.3)
-cset = ax.contour(X, Y, Z, zdir='z', offset=-100, cmap=cm.coolwarm)
+
+# Plot projections of the contours for each dimension.  By choosing offsets
+# that match the appropriate axes limits, the projected contours will sit on
+# the 'walls' of the graph
 cset = ax.contour(X, Y, Z, zdir='x', offset=-40, cmap=cm.coolwarm)
 cset = ax.contour(X, Y, Z, zdir='y', offset=40, cmap=cm.coolwarm)
+cset = ax.contour(X, Y, Z, zdir='z', offset=-100, cmap=cm.coolwarm)
```

Comment:
Do you mean that it's better to plot the z one first because it's on the 'bottom' from the perspective of the default angle, or something else?  And if so, do you think that choice warrants a comment?